docs/index.html

```diff
@@ -851,7 +851,7 @@ <h5 class="b-header">In remix tools</h5>
         </div>
         <div class="has-background-white pt-card-padding-small">
           <p class="text-normal  padding-bottom-small">Any automated attribution should include at a minimum the author and license of the original work (with link to the specific license). Best practice is to retain TASL, standing for Title, Author, Source, and License, in any attribution. </p>
-          <a href="https://wiki.creativecommons.org/wiki/Best_practices_for_attribution" rel=”noopener noreferrer” target="_blank"><button class="button pt-button-blue-lg pt-button-large">Best practice</button></a>
+          <a href="https://wiki.creativecommons.org/wiki/Best_practices_for_attribution" rel=”noopener noreferrer” target="_blank"><button class="button pt-button-blue-lg" id="pt-button-large">Best practice</button></a>
```

Comment:
IDs only exists once in a document because it works to identify elements, they are like unique identifiers.
you can read more about that [here](https://css-tricks.com/the-difference-between-id-and-class/)

Reply:
@hugosolar thankyou for the review, I have made appropriate changes by adding a class overflow to override the properties, and removed the !important as well. Please review.